static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -107,10 +109,29 @@ export default function FeedbackListPage() {
   );
 }
 
-const LayoutGrid = styled('div')`
+const Background = styled('div')`
   background: ${p => p.theme.background};
   overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  gap: ${space(2)};
+
+  @media (max-width: ${p => p.theme.breakpoints.medium}) {
+    padding: ${space(2)};
+  }
+
+  @media (min-width: ${p => p.theme.breakpoints.medium}) {
+    padding: ${space(2)};
+  }
+
+  @media (min-width: ${p => p.theme.breakpoints.large}) {
+    padding: ${space(2)} ${space(4)};
+  }
+`;
 
+const LayoutGrid = styled('div')`
+  overflow: hidden;
```

Comment:
Is this going to hide any tooltips/hovercards/etc?

Reply:
It'll cutoff tooltips and stuff if they go past the bounds of the grid. It was like that before too.

I tried a little to get rid of this, i think some combo of `contain:strict` and some other tricks will work. but it started to get out of scope for this change so i punted. The replay details page has the same kind of wrapper problem too, it cuts off those :focus rings.